development.ini

```diff
@@ -11,7 +11,7 @@ pyramid.default_locale_name = en
 
 jujugui.sandbox = true
 jujugui.raw = true
-jujugui.combine = true
+jujugui.combine = false
```

Comment:
Because this is true in production JC mentioned that we should leave it as combined by default so that we will notice any potentially issues sooner.